java/src/main/resources/template.properties

```diff
@@ -242,6 +242,20 @@ kafka.bq.fail.on.dataloss=false
 ## Ouptut mode for writing data. Accepted values: 'append', 'complete', 'update'
 kafka.bq.stream.output.mode=append
 
+
+#Kafka Common
+kafka.message.format=
```

Comment:
we need to follow consistent naming pattern for config variables like
`kafka.gcs.input.message.format`
And so on for all the inputs and outputs

Reply:
As discussed , the sink value is deliberately omitted as these are source related parameters and the reusable KafkaReader can be used for any Kafka based Template irrespective of Sink